package.json

```diff
@@ -40,13 +40,12 @@
     "@types/react": "^18.0.24",
     "@types/react-dom": "^18.0.8",
     "@types/w3c-web-usb": "^1.0.6",
-    "@typescript-eslint/eslint-plugin": "^5.42.1",
-    "@vitejs/plugin-react": "^2.2.0",
```

Comment:
What is the reason for removing `@vitejs/plugin-react`?
This plugin should not be removed because it is used in example/vite.config.ts.

![スクリーンショット 2023-10-22 12 07 37](https://github.com/8beeeaaat/web-ant-plus/assets/4495546/f35bb55e-0fcc-46f9-a5a0-e4ba75eebd66)

https://github.com/8beeeaaat/web-ant-plus/blob/7916f5cebdee2f3080e1437b62ca0837181e75ed/example/vite.config.ts#L1-L7

Reply:
Deleted by accident, re adding.